docs/pages/why-web-components.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Why web components
+description: Explanation of why we use web components in the CA design system
+keywords: 
+---
+
+
+## Component-based development
+
+Building sites with components has a lot of benefits for site visitors and the teams that run them.
+
+### How components work
+
+Components are bite-sized sections of code that link together to create a website. They&rsquo;re kind of like LEGO bricks. They&rsquo;re often uniform and reusable.
+
+Components help developers and designers save time. They help you avoid rewriting redundant code. You can also easily plug them into and remove them from code using HTML tags.
+
+The California Design System uses the minimal amount of tech required for each component. This is&nbsp;often just HTML &amp; CSS. When we need JavaScript, we take advantage of the component model built into the browser. We use the custom elements part of the web components specification. This allows us to define new HTML tags like &lt;cagov-accordion&gt; or &lt;cagov-page-feedback&gt;, which are inert until upgraded by provided JavaScript.
```

Comment:
Can we use code style for `<cagov-accordion>`  and `<cagov-page-feedback>`? Or do they get escaped?

Reply:
updated